src/data-publication/reports/ReportPagination.css

```diff
@@ -0,0 +1,78 @@
+.ReportPagination {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.ReportPagination.bottom {
```

Comment:
Might be over analyzing here but I believe we need to add a space between the end of the first class and the second class?

Reply:
No we don't want a space because `.bottom` is applied to the same element as `.ReportPagination`, a space between would mean that `.bottom` is a child of `.ReportPagination`.
 
 [code](https://github.com/cfpb/hmda-frontend/pull/1562/files#diff-faa7927f0bb85149b0130c4aac7009e20fbdbed49f0978265f97b075170d61e7R22-R26)